disputer/disputer.js

```diff
@@ -148,7 +148,7 @@ class Disputer {
 
       const txnConfig = {
         from: this.account,
-        gas: Math.min(gasLimit, this.txnGasLimit),
+        gas: Math.min(Math.floor(gasEstimation * this.GAS_LIMIT_BUFFER), this.txnGasLimit),
```

Comment:
Should we set `cost gasToSend = Math.min(Math.floor(gasEstimation * this.GAS_LIMIT_BUFFER), this.txnGasLimit)` right after we calculate the `gasEstimation`? Since we re-use this calculation twice

Reply:
AH nevermind, I see the `gasEstimations` are for different txns